### PR TITLE
[Merged by Bors] - feat port/CategoryTheory.Pi.Basic

### PR DESCRIPTION
**Pull request by mattrobball**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Pi/Basic.lean [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+using '#align category_theory.pi.sum_elim_category
+CategoryTheory.Pi.sumElimCategoryₓ'. -/
+instance sumElimCategory : ∀ s : Sum I J, Category.{v₁} (Sum.elim C D s)
+  | Sum.inl i => by
+    dsimp
+    infer_instance
+  | Sum.inr j => by
+    dsimp
+    infer_instance
+#align category_theory.pi.sum_elim_category CategoryTheory.Pi.sumElimCategory
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/Pi/Basic.lean:169]:
```suggestion
instance sumElimCategory : ∀ s : Sum I J, Category.{v₁} (Sum.elim C D s)
  | Sum.inl i => by
    dsimp
    infer_instance
  | Sum.inr j => by
    dsimp
    infer_instance
#align category_theory.pi.sum_elim_category CategoryTheory.Pi.sumElimCategoryₓ -- Porting note: reordered universes
```

Comment (mattrobball) [reply Mathlib/CategoryTheory/Pi/Basic.lean:169]: @jcommelin they look the same to me up to relabeling. I just wanted another set of eyes to confirm. 😆 

If you agree, I will delete all of this.

Comment (jcommelin) [reply Mathlib/CategoryTheory/Pi/Basic.lean:169]: I agree. But note that my suggestion adds the little `ₓ` to the align statement, as the comment instructs. And please leave a porting note.

**Review by jcommelin — COMMENTED** [review]
bors d+

File: Mathlib/CategoryTheory/Pi/Basic.lean (hunk 1 above)
Comment (mattrobball) [reply Mathlib/CategoryTheory/Pi/Basic.lean]: Done.

**Comment by mattrobball** [comment]
bors merge

**Pull request closed without merging**